### PR TITLE
Beryllium lnwallet with alex stuff as well 

### PR DESCRIPTION
**Pull request by eoliveros**

### CODE REVIEW

File: ln.py [hunk 1]
```diff
@@ -148,6 +148,13 @@ def multi_withdraw(self, outputs_dict):
     def prepare_psbt(self, outputs):
         return self.instance.txprepare(outputs)
 
+    def send_invoice(self, bolt11):
+        # pay a bolt11 invoice
+        invoice_result = self.instance.pay(bolt11)
+        invoice_result["sats_sent"] = int(
```
Comment (djpnewton) [inline ln.py:154]: use `_msat_to_sat()`

File: templates/layout.html [hunk 2]
```diff
@@ -78,7 +83,32 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/bootbox.js/5.5.2/bootbox.min.js" integrity="sha384-x8yg1bbOwuqjTf+DF/Sc3v14Tfzp2C61ThwPbDgflWBra3aBH4/vrB4z1WH3WwpP" crossorigin="anonymous"></script>
-
+    <script type="text/javascript" charset="utf-8">
```
Comment (djpnewton) [inline templates/layout.html:86]: this should be inside the `scripts` block on just the template that requires it

File: templates/layout.html [hunk 3]
```diff
@@ -7,6 +7,11 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <!-- Socket.IO JS -->
```
Comment (djpnewton) [inline templates/layout.html:10]: this should be inside the scripts block on just the template that requires it

File: templates/lightning/layout.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```
Comment (djpnewton) [inline templates/lightning/layout.html:1]: if this file is not used then get rid of it

File: templates/lightning/template.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{% extends "layout.html" %}
```
Comment (djpnewton) [inline templates/lightning/template.html:1]: i dont think this file is used?

File: web.py [hunk 6]
```diff
@@ -75,6 +79,24 @@ def process_deposits_and_broker_orders():
...
+#    """ Returns SVG of input data """
+#    factory = qrcode.image.svg.SvgPathImage
+#    img = qrcode.make(data, image_factory=factory, box_size=35)
+#    output = io.BytesIO()
+#    img.save(output)
+#    svg = output.getvalue().decode('utf-8')
+#    return svg
+#
+def qrcode_svg_create_ln(data):
+    """ Returns SVG of input data (LN focused) """
```
Comment (djpnewton) [inline web.py:92]: how is this function ln focused? the box size? just add box_size as a parameter to the `qrcode_svg_create` function

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: ln.py (hunk 1 above)
Comment (eoliveros) [reply ln.py]: changed

File: templates/lightning/layout.html (hunk 4 above)
Comment (eoliveros) [reply templates/lightning/layout.html]: removed this page

File: templates/lightning/template.html (hunk 5 above)
Comment (eoliveros) [reply templates/lightning/template.html]: removed this page.

File: web.py (hunk 6 above)
Comment (eoliveros) [reply web.py]: import qrcode_svg_create from utils and used that instead of qrcode_svg_create_ln

File: templates/lightning/generic.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
```
Comment (djpnewton) [inline templates/lightning/generic.html:1]: this file not used?

File: templates/layout.html (hunk 3 above)
Comment (djpnewton) [reply templates/layout.html:10]: this still not addressed

File: templates/layout.html (hunk 2 above)
Comment (djpnewton) [reply templates/layout.html:86]: this still not addressed

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: templates/lightning/generic.html (hunk 7 above)
Comment (eoliveros) [reply templates/lightning/generic.html:1]: i thought i remove this. having a look again

Comment (eoliveros) [reply templates/lightning/generic.html:1]: nope, missed it. ill delete this line after the push from Alex

File: templates/layout.html (hunk 2 above)
Comment (eoliveros) [reply templates/layout.html:86]: waiting on alex for this.

File: templates/layout.html (hunk 3 above)
Comment (eoliveros) [reply templates/layout.html:10]: waiting on alex for this.

**Pull request merged**